src/KnowledgeQuiz.tsx

```diff
@@ -42,175 +42,143 @@ const fieldsetStyle = css`
+                            className={css`
+                                padding-right: 12px;
+                            `}
+                        >
+                            {number + '.'}
+                        </span>
+                        {text}
+                    </legend>
+                </div>
+                {imageUrl && (
+                    <img
+                        className={css`
+                            width: 100%;
+                        `}
+                        src={imageUrl}
+                    />
+                )}
+                <div
                     className={css`
-                        width: 100%;
+                        /* fix for chrome jumping to top of page */
+                        position: relative;
                     `}
-                    src={imageUrl}
-                />
-            )}
-            <div
-                className={css`
-                    /* fix for chrome jumping to top of page */
-                    position: relative;
-                `}
-            >
-                <Answers
-                    id={id}
-                    answers={answers}
-                    hasSubmittedAnswers={hasSubmittedAnswers}
-                    selectedAnswers={selectedAnswers}
-                    updateSelectedAnswer={updateSelectedAnswer}
-                />
-            </div>
-        </fieldset>
-    </div>
-);
+                >
+                    <Answers
+                        id={id}
+                        answers={answers}
+                        hasSubmitted={hasSubmitted}
+                        selectedAnswer={selectedAnswer}
+                        updateSelectedAnswer={updateSelectedAnswer}
+                    />
+                </div>
+                <div
+                    className={css`
+                        display: flex;
+                        flex-direction: row;
+                        button {
+                            margin-right: 10px;
+                        }
+                    `}
+                >
+                    <Button
+                        type="submit"
+                        data-testid={`submit-question-${id}`}
+                        onClick={(e) => {
+                            e.preventDefault();
+                            setHasSubmitted(true);
+                        }}
+                        onKeyDown={(e) => {
+                            const spaceKey = 32;
+                            const enterKey = 13;
+                            if (
+                                e.keyCode === spaceKey ||
+                                e.keyCode === enterKey
+                            ) {
+                                // prevent submit
+                                e.preventDefault();
+                                setHasSubmitted(true);
+                            }
+                        }}
+                    >
+                        Submit
```

Comment:
I'm still not sure the word Submit is correct here. Maybe 'check' or 'reveal'? I think having multiple Submit buttons on one page is maybe a bit semantically, er, well, I guess it just feels off?